python/UniverseWrapper.cpp

```diff
@@ -76,9 +76,10 @@ namespace {
     std::vector<int> ObjectIDs(const Universe& universe)
     {
         std::vector<int> result;
-        result.reserve(universe.Objects().size());
-        for (auto it = universe.Objects().begin<T>(); it != universe.Objects().end<T>(); ++it)
-            result.push_back(it->ID());
+        auto objs = universe.Objects().all<T>();
```

Comment:
Is there a purpose or need for storing the result of calling all in a local rather than putting the all call into the loop declaration directly?

Reply:
I usually do those mass substitutions via `sed` and let the compiler decide if the change was good or not.  When I see some place for optimization when I check in I usually do so, but I miss sometimes stuff.

Reply:
I assumed something like that, but thought I should ask in case there was a reason for it still.

Reply:
See d9fb4f9bf58fb845bf4547dd4b9e1f53bc7df8e0